### PR TITLE
BigNumber

### PR DESCRIPTION
**Pull request by JeremyBernier**
image

### CODE REVIEW

File: react/src/CanvasElement.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+export const CanvasElement = ({
+    element,
+    elementId,
+    dataHash,
+}: {
+    element: EmbedElement;
+    elementId: string;
+    dataHash: string;
+}) => {
+    if (!element) return <></>;
```
Comment (wpride) [inline react/src/CanvasElement.tsx:17]: type for `element` should be allow for undefined

File: react/src/CanvasElement.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                />
+            </Element>
+        );
+    }
+    if (elementType.type === 'component') {
+        if (elementType.component.component === 'BigNumber') {
+            return <BigNumber element={element} />;
+        }
+    }
+    return <></>;
```
Comment (wpride) [inline react/src/CanvasElement.tsx:53]: Maybe a `console.log` here for unrecognized element type?

File: react/src/components/BigNumber.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const BigNumber = ({ element }) => {
```
Comment (wpride) [inline react/src/components/BigNumber.tsx:3]: Can we have types here?

File: react/src/components/BigNumber.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    // TODO: this px-7 padding should be consolidated with Element.tsx
+    return (
+        <div className="px-7">
+            <h3 className="text-[15px] font-medium text-default/80 mb-1">{element.title}</h3>
+            <div className="text-3xl">{element.elementType.grid.data[1].value.formatted_value}</div>
```
Comment (wpride) [inline react/src/components/BigNumber.tsx:8]: Should check that `data[1]` exists

**Review by wpride — APPROVED** [review]
Fuck yeah 🔥 

Big comment is just about `Cell` - its probably overkill to send the entire `Cell` down since we just want strings here.

Higher level, would be good to have `typescript` and `eslint` CI here since there's at least one unsafe array access in here

File: react/src/CanvasElement.tsx (hunk 2 above)
Comment (JeremyBernier) [reply react/src/CanvasElement.tsx:56]: added

File: react/src/CanvasElement.tsx (hunk 1 above)
Comment (JeremyBernier) [reply react/src/CanvasElement.tsx:18]: updated element prop type to allow undefined

File: react/src/components/BigNumber.tsx (hunk 3 above)
Comment (JeremyBernier) [reply react/src/components/BigNumber.tsx]: added

File: react/src/components/BigNumber.tsx (hunk 4 above)
Comment (JeremyBernier) [reply react/src/components/BigNumber.tsx]: updated

**Comment by JeremyBernier** [comment]
> Higher level, would be good to have `typescript` and `eslint` CI here since there's at least one unsafe array access in here

Definitely. I did copy over the .eslintrc.js from the old project. Maybe need to add that to the CI/CD.

**Review by wpride — APPROVED** [review]
Fuego

**Pull request merged**